settings.gradle.kts

```diff
@@ -1,330 +1,5 @@
-    if (artifactoryUrl.isNullOrBlank()) {
-        name = repositoryName
-        setUrl(originalRepo)
-    } else {
-        name = "Proxy for $repositoryName: $originalRepo"
-        artifactoryUrl(repositoryName)
-    }
-}
-
-@Suppress("UnstableApiUsage")
-dependencyResolutionManagement {
-    repositories {
-        maven {
-            setUrlOrProxy("mavenCentral", "https://repo1.maven.org/maven2")
-        }
-
-        // not available in mavenCentral
-        exclusiveContent {
-            forRepository {
-                maven {
-                    setUrlOrProxy("jcenter", "https://jcenter.bintray.com")
-                }
-            }
-            filter {
-                includeGroup("org.funktionale")
-                includeGroup("org.jetbrains.trove4j")
-                includeGroup("com.forkingcode.espresso.contrib")
-                includeModule("org.jetbrains.teamcity", "teamcity-rest-client")
-                includeModule("com.fkorotkov", "kubernetes-dsl")
-                includeModule("me.weishu", "free_reflection")
-            }
-        }
-
-        // for kotlinx-cli https://github.com/Kotlin/kotlinx-cli/issues/23
-        exclusiveContent {
-            forRepository {
-                maven {
-                    setUrlOrProxy("KotlinX", "https://kotlin.bintray.com/kotlinx")
-                }
-            }
-            filter {
-                includeGroup("org.jetbrains.kotlinx")
-            }
-        }
-        exclusiveContent {
-            forRepository {
-                maven {
-                    setUrlOrProxy("google-android", "https://dl.google.com/dl/android/maven2/")
-                }
-            }
-            filter {
-                includeModuleByRegex("com\\.android.*", "(?!r8).*")
-                includeModuleByRegex("com\\.google\\.android.*", ".*")
-                includeGroupByRegex("androidx\\..*")
-                includeGroup("com.google.test.platform")
-                includeGroup("com.google.testing.platform")
-            }
-        }
-        exclusiveContent {
-            forRepository {
-                maven {
-                    setUrlOrProxy("r8-releases", "http://storage.googleapis.com/r8-releases/raw")
-                }
-            }
-            filter {
-                includeModule("com.android.tools", "r8")
-            }
-        }
-    }
-}
-
-plugins {
-    id("com.gradle.enterprise") version "3.5.1"
-}
-
-val isCI = booleanProperty("ci", false)
-
-gradleEnterprise {
-    buildScan {
-        termsOfServiceUrl = "https://gradle.com/terms-of-service"
-        termsOfServiceAgree = "yes"
-        publishOnFailureIf(isCI)
-    }
-}
-
-fun booleanProperty(name: String, defaultValue: Boolean): Boolean {
-    return if (settings.extra.has(name)) {
-        settings.extra[name]?.toString()?.toBoolean() ?: defaultValue
-    } else {
-        defaultValue
-    }
-}
+includeBuild("subprojects")
```

Comment:
What negative sides do you see?

For instance:
- Will any task rebuild all subprojects?

Reply:
nope, it won't 
`Task 'assemble' not found in root project 'avito-android-infra'.`